### DISCUSSION TITLE
Do we really still need CCM in the playbooks?

### DISCUSSION THREAD

**Discussion by marde16** [HCL-TECH-SOFTWARE/connections-automation#299] [Q&A]
CCM is end of life 

[End of Support for Select HCL Connections Portfolio Product Versions and Products](https://support.hcltechsw.com/csm?id=kb_article&sysparm_article=KB0095999)

.. but the playbook contains CCM.

https://github.com/HCL-TECH-SOFTWARE/connections-automation/blob/97eb462cd579cbb2f8accb63e22c7e81e5e3bcc4/roles/hcl/connections/tasks/install_connections.yml#L17-L27

Do we really still need this?

**Comment by sabrina-yee** [comment]
Apologies for the late reply.  
No it's not needed.  These steps will be skipped since `ccm_install` is not defined.
